src/pages/SettingsPage.js

```diff
@@ -72,6 +72,22 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
     if (!ValidUrl.isWebUri(newSyncURL)) ToastAndroid.show('Not a valid URL', ToastAndroid.LONG);
     else setState({ ...state, modalKey: '', syncURL: newSyncURL });
   };
+
+  const onCheckConnection = () => {
+    //  Hash the password.
+    const AUTH_ENDPOINT = '/sync/v3/site';
```

Comment:
Can we put this strings in the file: `src/authentication/index` or export it from `SyncAuthenticator` - that way, we only have the string in one place.

Reply:
Fixed by this: 3ab1539